modules/cuda_plugin/include/cuda/cuda_config.hpp

```diff
@@ -51,7 +51,7 @@ DECLARE_CUDA_CONFIG_VALUE(YES);
 DECLARE_CUDA_CONFIG_VALUE(NO);
 
 /**
- * @brief Defines the number of throutput streams used by CUDA plugin.
+ * @brief Defines the number of throutput streams used by NVIDIA GPU plugin.
```

Comment:
should we rename the folder with plugin and all files as well?

Reply:
It is draft PR, it is not yet ready for review that is why I added [WIP].
Yes, folder will be renamed

Reply:
@ilya-lavrenov Lets me clarify something, what do you mean by renaming also files ?
Do you mean about renaming files with prefix `cuda_*` ?

Reply:
@ilya-lavrenov PR is ready for review

Reply:
Do we need to make CI green first?

Reply:
@ilya-lavrenov Yes, but we need to merge first PR in openvino. Because these PRs have cross-dependecies we need to figured out the order of merge. Please, see https://github.com/openvinotoolkit/openvino/pull/12719